tests/test-buffer.js

```diff
@@ -84,4 +84,13 @@ assert.throws(function () {
     buff.toString("utf8");
 }, "Error thrown when 'hex' is not given to toString()");
 
+var buff = new Buffer(4);
+var writeValue = -0.232;
+// write negative float value (14 bit precision) as uint32 by disabling noAssert.
+buff.writeUInt32BE((writeValue * Math.pow(2, 32-14)) << 1, 0, true);
+var readValue = (buff.readUInt32BE(0) >> 1) / Math.pow(2, 32-14);
+assert(writeValue.toPrecision(3) === readValue.toPrecision(3),
+       "The value of readUInt32BE() with precision 3 expected: "
+       + writeValue + " got: " + readValue.toPrecision(3));
```

Comment:
Hmm this doesn't work on A101 because we drop support for Math libraries and such to fit. But I guess it works in jslinux. We can probably leave it in and work on this in the future.